cmd/syft/cli/options/catalog.go

```diff
@@ -77,6 +158,20 @@ func (cfg *Catalog) AddFlags(flags clio.FlagSet) {
 	flags.StringArrayVarP(&cfg.Catalogers, "catalogers", "",
 		"enable one or more package catalogers")
 
+	if pfp, ok := flags.(fangs.PFlagSetProvider); ok {
+		if err := pfp.PFlagSet().MarkDeprecated("catalogers", "use: override-default-catalogers and select-catalogers"); err != nil {
+			panic(err)
+		}
+	} else {
+		panic("unable to mark flags as deprecated")
+	}
+
+	flags.StringArrayVarP(&cfg.DefaultCatalogers, "override-default-catalogers", "",
+		"set the catalogers to use explicitly as the final cataloger set (defaults to 'image' or 'directory' dynamically depending on the scan source)")
+
+	flags.StringArrayVarP(&cfg.SelectCatalogers, "select-catalogers", "",
+		"add to, remove from, or select a subset of catalogers from the final cataloger set")
```

Comment:
I'd like to talk about the wording of this a bit. Maybe "*for* the final cataloger set" instead of "*from* the final cataloger set"?

Reply:
I'll take @kzantow s wording here